### ISSUE TITLE
Implement a Query Builder similar to SQL Developer

### ISSUE THREAD

**Issue by g-r-u** [OmniDB/OmniDB#76]
This feature would expand Omni's user-base to non-technical users or users with no SQL experience.
[https://docs.oracle.com/cd/B25329_01/doc/appdev.102/b25309/qry_bldr.htm](Oracle Docs - Building Queries with Query Builder).

**Comment by wind39** [comment]
Yes, it would be very useful, thank you for your suggestion! :+1:
